script/testing/jdbc/src/PelotonTest_Lin.java

```diff
@@ -0,0 +1,462 @@
+import java.sql.*;
+import org.postgresql.util.*;
+
+
+/**
```

Comment:
Why do we need two testing files `PelotonTest_Lin.java` ? 

Also, remove names and dates. Instead, please add the license from some other file.